### PR TITLE
Update temporarily deleted shop routes

### PR DESCRIPTION
**Pull request by kevinkoech357**
​
## Description
Update the routes to get all temporarily deleted shopsand also their details based on ID to follow API naming conventions and  best practices.
Also updated swagger docs to reflect the same.
​
## Related Issue (Link to linear ticket)
None
​
## Motivation and Context
Follow API naming conventions and best practices.
​
## How Has This Been Tested?
N/A
​
## Screenshots (if appropriate - Postman, etc):
​
## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
​
## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: super_admin_1/shop/routes.py [hunk 1]
```diff
@@ -774,14 +776,18 @@ def perm_del(user_id, shop_id):
...
+def get_temporarily_deleted_vendors(user_id: UUID, status: str = "soft-deleted") ->  jsonify:
     """
     Retrieve temporarily deleted vendors.
 
     This endpoint allows super admin users to retrieve a list of vendors who have been temporarily deleted.
 
+    Args:
+        user_id (uuid): The ID of the super admin user making the request.
+        status (str, optional): The status of the vendors to retrieve. Default: "soft-deleted".
+
```
Comment (zxenonx) [inline super_admin_1/shop/routes.py:790]: Good

**Comment by zxenonx** [comment]
I had already suggested modifying this route /temporarily-deleted-..... route, but this was not done.
Let the FE use what's already there then.
@kevinkoech357

**Comment by Freeman-kuch** [comment]
@zxenonx this has been done in my PR #197

**Comment by kevinkoech357** [comment]
@zxenonx Ok.

**Pull request closed without merging**